File: discovery-service/src/main/java/com/sudo248/discoveryservice/controller/dto/SupplierProductInfoDto.java
package com.sudo248.discoveryservice.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SupplierProductInfoDto {
    private String supplierId;
    private String productId;
    private String categoryId;
    private List<ImageDto> productImages;
    private String description;
    private String productName;
    private String categoryName;
    private int amountLeft = 0;
    private double price = 0.0;
    private int soldAmount = 0;
    private double rate = 0.0;
    private String sku;
}

File: client/SocStaff/app/src/main/java/com/sudo248/soc_staff/data/dto/discovery/SupplierProductInfoDto.kt
package com.sudo248.soc_staff.data.dto.discovery

data class SupplierProductInfoDto(
    val supplierId: String = "",
    val productId: String = "",
    val categoryId: String = "",
    val productImages: List<ImageDto> = listOf(),
    val description: String = "",
    val productName: String = "",
    val categoryName: String = "",
    val amountLeft: Int = 0,
    val price: Double = 0.0,
    val soldAmount: Int = 0,
    val rate: Double = 0.0,
    val sku: String = "",
)
